client/assets/stylesheets/common/_variables.scss

```diff
@@ -4,3 +4,6 @@ $font-weight-regular: 400;
 $font-weight-medium: 500;
 $font-weight-semibold: 600;
 $font-weight-bold: 700;
+
+// Font family
+$font-family-icons: "Material Design Icons";
```

Comment:
Technically we could end up using multiple icon fonts so I would rename it to `font-family-icons`

Reply:
Since you extracted icons font to a variable you can do the same with others.
As far as I see we have two options:
1. `Poppins, Helvetica, Arial, sans-serif` as the primary font and
2. `Roboto, Helvetica, Arial, sans-serif` as secondary.